src/wirecraft/client/server_interface.py

```diff
@@ -12,8 +38,23 @@ class ServerInterface:
     vers un serveur distant sans trop de refactoring.
     """
 
-    def __init__(self) -> None:
-        self.connection = Server()
+    def __init__(self, game: Game) -> None:
+        self.game = game
+        self.connection = Server(self)
+        self.connection.start()
+
+    def close_connection(self):
+        self.connection.stop()
 
     def get_money(self) -> int:
+        # The client can ask for data to the server
```

Comment:
Should we prefix the function name to differentiate client → server from server→ client ?

Reply:
server -> client functions are designed with the @event decorator 

This decorator is just "cosmetic" but also allows us to differentiate client -> server methods from server -> client